src/lib/y2storage/blk_device.rb

```diff
@@ -382,11 +413,11 @@ def part_of_lvm_or_md?
     # (directly or through an encryption) or any RAID having this device as
     # one of its members.
     #
-    # @return [Array<Device>] a collection of MD RAIDs, DM RAIDs, volume groups
-    #   and multipath devices
+    # @return [Array<Device>] a collection of MD RAIDs, DM RAIDs, volume groups,
+    #   multipath, bcache and bcache_cset devices
     def component_of
       vg = lvm_pv ? lvm_pv.lvm_vg : nil
-      (dm_raids + [vg] + [md] + [multipath]).compact
+      (dm_raids + [vg] + [md] + [multipath] + [bcache] + [direct_bcache_cset]).compact
```

Comment:
There is no `BcacheCset#name`, so make sure to handle that case in `#component_of_names`,